ocs_ci/helpers/sanity_helpers.py

```diff
@@ -31,7 +39,8 @@ def __init__(self):
         self.pod_objs = list()
         self.obc_objs = list()
         self.obj_data = ""
-        self.ceph_cluster = CephCluster()
+        if not is_hci_client_cluster():
+            self.ceph_cluster = CephCluster()
```

Comment:
ceph_cluster attribute looks like an important part of Sanity class. It is used for health_check for ceph and noobaa. This could make discrepancies for some functions like create_resources and health_check.

Reply:
When trying to run this code on a client, it couldn't get the client `Cephcluster` here: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/cluster.py#L90, as it doesn't have any.

Reply:
Ok, there will be probably some issues with health_check and validation of some resources in some tests but it can be taken care of in a different PR when those issues are discovered. You can add a note that this is missing so it is known when debugging.
In future there can be a new cluster class similar to `CephClusterExternal` (https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/cluster.py#L2378) that can be used on client and takes care of proper context (storage cluster and health checks are accessed on provider).

Reply:
Okay, that makes sense. I create a new issue for tracking and implementing in the future: https://github.com/red-hat-storage/ocs-ci/issues/9574.